config/rewrite.yml

```diff
@@ -1,6 +1,6 @@
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.checkstyle.AllAutoFixes
+name: org.checkstyle.openrewrite.SanityCheck
```

Comment:
`org.checkstyle.openrewrite.AllAutoFixes`

Reply:
yes lets see it then as impl detail. Still the topic is a generic sanity check imho. will obey.

- https://github.com/checkstyle/checkstyle/pull/18430